libsolidity/formal/ModelChecker.cpp

```diff
@@ -125,7 +125,12 @@ void ModelChecker::analyze(SourceUnit const& _source)
 	if (m_settings.engine.chc)
 		m_chc.analyze(_source);
 
-	auto solvedTargets = m_chc.safeTargets();
+	map<ASTNode const*, set<VerificationTargetType>, smt::EncodingContext::IdCompare> solvedTargets;
+
+	for (auto const& [node, targets]: m_chc.safeTargets())
```

Comment:
Given that we already use `rangev3`, would something like
```c++
for (auto const& target : ranges::views::values(m_chc.safeTargets())
{
  ...
}
```
work?

Reply:
Oh, it wouldn't, because you need `node` as well.

Reply:
Yep